fhirResourcesToLoad/o2billy_03__observation_hco3.json

```diff
@@ -0,0 +1,65 @@
+{
+  "resourceType": "Observation",
+  "id": "obs015-hco3",
+  "status": "final",
+  "contained": [
```

Comment:
Why do we need contained resource? It is not very common having contained practitioner/organization

Reply:
The original ideas was to use the contained value as we can't retrieve the practitioner and organization which are not directly related to the DeviceRequest. Yeah, you are right. That is not the EHR limitation instead of our software's limitation. I removed the contained part and am using the real reference now. If in the future, we change DTR to get more FHIR resources, we would be able to retrieve then populate the fields.